provider/github/watcher_test.go

```diff
@@ -155,6 +155,30 @@ func (s *WatcherTestSuite) TestWatch() {
 	}
 }
 
+func (s *WatcherTestSuite) TestWatchDraftPR() {
+	called := false
+	ctx, cancel := context.WithCancel(context.Background())
+	s.mux.HandleFunc("/repos/mock/test/pulls", func(w http.ResponseWriter, r *http.Request) {
+		if called {
+			cancel()
+		}
+		fmt.Fprint(w, `[{"id":1},{"id":2,"draft":true},{"id":3}]`)
```

Comment:
Wound't it be better to use lookout-test-fixtures here?

Reply:
1. It's a unit test, usually, it's a bad idea to use huge fixtures in unit tests. Unit tests should test the smallest part of the logic possible and it should be easy to understand what they do.
2. We don't have fixtures for PRs list in lookout-test-fixtures repository.

Reply:
I don't entirely agree. In my opinion using fixture data does not automatically transform a unit test into an integration test. In this case the fixture is even vendored, so it's not so different from a hardcoded string.

But I don't want to block this PR for this.